### ISSUE TITLE
Error en el sandbox de suscripciones

### ISSUE THREAD

**Issue by walis85300** [mercadopago/DEPRECATED-sdk-php#112]
Estoy probando el sandbox de suscripciones de mercadopago, puedo redirigir hacia el sitio donde se me listan unas tarjetas, donde están varios errores disponibles, las primeras son de pago aprobado, pero después de darle presionar pago automático el sandbox muestra un error de que algunos de los datos es erróneo.

![screen shot 2018-04-03 at 16 56 13](https://user-images.githubusercontent.com/4245976/38275473-00e4655c-3760-11e8-91c4-e86307966b97.png)

![screen shot 2018-04-03 at 16 56 18](https://user-images.githubusercontent.com/4245976/38275478-04b8b55c-3760-11e8-8630-d0df8cd3fc25.png)

No tengo idea si me estoy equivocando en alguno de los campos que estoy enviando, las credenciales son de usuarios de prueba generados con mi cuenta, ambos, comprador y vendedor son de México.

Este es mi código: 

`$preapproval_data = array(
            "external_reference" => "UnaPrueba",
            "payer_email" => Auth::user()->email,
            "back_url" => route('pagos.exitoso'),
            "reason" => $plan->descripcion,
            "auto_recurring" => array(
                "frequency" => 1,
                "frequency_type" => "months",
                "transaction_amount" => $plan->precio,
                "currency_id" => "MXN",
                "start_date" => "2018-04-03T20:58:11.778-03:00",
                "end_date" => "2019-04-03T20:58:11.778-03:00",
            )
        );`

El desarrollo de la aplicación está detenido por este problema, si no consigo una solución esta semana deberé cambiarme de procesador de pagos.

**Comment by garciaeric** [comment]
Efectivamente, el sandbox de suscripciones no esta funcionando hace bastante tiempo.
Para hacer este tipo de test tenes que generar un usuario tester comprador y vendedor utilizando las API de MercadoPago.
Saludos!
